### ISSUE TITLE
Stuck on Apple logo after install Appsync

### ISSUE THREAD

**Issue by kirby1x** [pwn20wndstuff/Undecimus#721]
**Describe the bug**
I have installed latest beta of uncover (41) jailbreak Ok all work fine, but when I installed "appsync unified" my iphone automaticly reboot, and now have Apple logo stuck on black screen.

I have iphone 6 (12.1), already tried to force reboot (home+power) no result.
Apple just stop signing 12.1.1 b3 today, no chance to restore my iphone and rejailbreak it.

**Device (please complete the following information):**
 - iOS Version: 12.1
 - iOS Device: Iphone 6
 - unc0ver Version: beta 41

**Place an "x" between the brackets if true:**
 - [ ] this is a bug others will be able to reproduce
 - [ ] this issue is present with all tweaks uninstalled(except for default packages) or disabled
 - [ ] this issue is present after a rootfs restore
 - [x] this issue is present on the latest version of unc0ver

**Logs**
If applicable, add logs or error messages here.

**Comment by iM4CH3T3** [comment]
Was it the first tweak u installed?

**Comment by kirby1x** [comment]
> Was it the first tweak u installed?

I dont remember, but I have already used 3 or 5 official tweaks from bigboss without any problem.
yesterday I have updated Uncover, all work fine, on the final stage I select appsync and install it.

Some icones like : options, mails, cydia, ... desapears from iphone, I have try shudown and reboot.
after that apple logo stuck on screen.

Any idea to solve this problem ?
